src/pages/about-netbird/faq.mdx

```diff
@@ -9,7 +9,11 @@ NetBird's agent doesn't require any incoming port to be open; It negotiates the
 ### Outgoing ports
 NetBird usually won't need open ports, but sometimes you or your IT team needs to secure and verify
 all outgoing traffic, and that may affect how NetBird clients connect to the [control layer](/about-netbird/how-netbird-works)
-and negotiate the peer-to-peer connections.
+and negotiate the peer-to-peer connections. 
+
+:unlock: Allowing the outbound **Relay (TURN)** service below is **recommended** for reliable peer connections. This will also improve the reliability of your [High Availability Routes](https://docs.netbird.io/how-to/routing-traffic-to-private-networks#high-availability-routes). 
+
+:unlock: If using `fail2ban` or similar also whitelist each netbird.io endpoint below.
```

Comment:
```suggestion
<Note>
    Allowing the outbound **Relay (TURN)** service below is **recommended** in more restricted networks for reliable peer connections. This will also improve the reliability of your [High Availability Routes](https://docs.netbird.io/how-to/routing-traffic-to-private-networks#high-availability-routes).
</Note>
<Note>
    If using `fail2ban` or similar, you should whitelist each netbird.io endpoint below.
</Note>
```